pkgs/development/libraries/libax25/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+, glibc
+, static ? stdenv.hostPlatform.isStatic
```

Comment:
You can do stdenv.hostPlatform.isStatic inline. I don't see the value for this parameter.

Reply:
Initially I was concerned with some downstream package's dependency on`libax25` and requiring it to be built statically vs not to satisfy the build dependencies. However, it doesnt look like thats a hard requirement (https://github.com/la5nta/pat/blob/master/make.bash#L36-L54) and inlining `stdenv.hostPlatform.isStatic` should be fine.